.github/workflows/devbuild.yml

```diff
@@ -123,7 +123,7 @@ jobs:
       run: |
         sudo pip3 install setuptools
         sudo pip3 install numpy pytest flake8
-        sudo pip3 install pyside6 shiboken6
+        sudo pip3 install pyside6==$(qmake6 -query QT_VERSION)
```

Comment:
OS package manager Qt version may not match with default version of pip installed pyside6, therefore need to specific version of pyside6 when using pip install.